index.html

```diff
@@ -0,0 +1,150 @@
+                                <h5 class="mb-0">
+                                    <button class="btn btn-link collapsed" data-toggle="collapse"
+                                        data-target="#TextToHex" aria-expanded="false" aria-controls="collapseTwo">
+                                        Text to Hex
+                                    </button>
+                                </h5>
+                            </div>
+                            <div id="TextToHex" class="collapse" aria-labelledby="headingTwo" data-parent="#accordion">
+                                <div class="card-body">
+                                    <div class="form-group green-border-focus">
+                                        <label for="hexInput">Hex Value:</label>
+                                        <textarea class="form-control rounded-0" rows="5" id="textInput" value=""
+                                            placeholder="Text input..."></textarea>
+
+                                        <div class="row d-flex align-items-center mb-3 mt-4">
+
+
+                                            <div class="col-md-12">
+                                                <div class="text-left">
+                                                    <button type="button"
+                                                        onclick="convertText(document.getElementById('textInput').value)"
+                                                        class="btn btn-outline-success btn-rounded z-depth-1a">Convert</button>
+                                                </div>
+                                            </div>
+
+
+                                        </div>
+
+                                        <label for="textOutput">Text Output:</label>
+                                        <textarea class="form-control rounded-0" rows="5" id="hexOutput" value=""
+                                            disabled placeholder="Text Output..."></textarea>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
+        integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
+        crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+        integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+        crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
+        integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
+        crossorigin="anonymous"></script>
+    <script>
+
+        const hexInput = document.querySelector('#hexInput');
```

Comment:
I like the code you used. It reminds me that I should use Regular Expressions especially with the type of calculator you used. 

The only thing I would say is I think it would be a lot better if you separate the HTML, CSS, and JavaScript files. You can link each file in the HTML. You can also publish your site on GitHub rather than use CodePen as well. I think it's best practice to separate them into 3 files.  Other than that great job!

Reply:
Thank you! I was actually wondering how to publish it on GitHub, I will have look into that. Yes its annoying copy it to CodePen :-)

Reply:
If you need any help on how to do it, DM me over on Discord. I will be happy to help you.